js/modules/treemap.src.js

```diff
@@ -609,27 +657,46 @@ seriesType(
                 this.axisTypes = colorSeriesMixin.axisTypes;
             }
 
-            Series.prototype.init.call(series, chart, options);
+            addEvent(series, 'setOptions', function (event) {
+                var options = event.userOptions;
 
-            options = series.options;
+                // Test if deprecated option is set.
+                if (defined(options.allowDrillToNode)) {
+                    error(
```

Comment:
1. Export text of error to number of error (like error 19/25).
2. Add chart context (third param of error).

Example:
```
H.error(18, true, chart);
```

It will allow us to keep full compatibility with `debug module`.

Reply:
See https://github.com/highcharts/highcharts/pull/9976#discussion_r250987773